src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/InfiniteScrollList.xaml.cs

```diff
@@ -717,5 +718,12 @@ private void _loadingStatusBar_DismissClick(object sender, RoutedEventArgs e)
         {
             _loadingStatusBar.Visibility = Visibility.Hidden;
         }
+
+        private void InfinitScrollListUnloaded(object sender, RoutedEventArgs e)
```

Comment:
typo 'Infinit**e**ScrollListUnloaded'

Reply:
Ok. I'll fix it. But main topic is what is causing this memory leak.